drain-cleaner/modules/drain-cleaner/module.yaml

```diff
@@ -8,7 +8,7 @@ envs:
     value: "/opt/strimzi"
 
 artifacts:
-  - md5: 6ac14ff4837d8ec65a3377bebf48dd56
+  - md5: 869c61ac2d8a8c18ca404e4c2728d96a
```

Comment:
this is wrong, the drain cleaner build didn't change you picked the SHA from the runner jar. Just revert back this change.
Just to make it clear, you didn't change the licences tar which is right. I mean, it should never happen that an MD5 is changed (jar) and not the other (license). So it was an alert, something was wrong.

Reply:
Thanks Paolo. Now reverted